src/Kushy/SymbolLoader.cs

```diff
@@ -284,12 +285,12 @@ private async Task<GlobalState> AddReferencedDatabasesAsync(GlobalState globals,
                 if (cluster != null)
                 {
                     // look for existing database of this name
-                    var db = cluster.Databases.FirstOrDefault(m => m.Name == dbRef.Database);
+                    var db = cluster.Databases.FirstOrDefault(m => m.Name == dbRef.Database.ToLower());
```

Comment:
Why are you ToLowering the database names?   Use a case insensitive string compare instead.

Reply:
changed.